README.md

```diff
@@ -6,6 +6,10 @@ Menu
 * Wings
 * Onion Rings
 
+## Romo Soups
+
+* Beef Tripe
```

Comment:
I think using `Romo` is derogatory and perhaps should be the full `Romanian` in the menu.